File: app/src/main/java/pe/andy/bookholic/ui/SearchDoneSnackBar.java
package pe.andy.bookholic.ui;

import com.google.android.material.snackbar.Snackbar;

import pe.andy.bookholic.databinding.MainActivityBinding;

public class SearchDoneSnackBar {

    MainActivityBinding mBinding;

    public SearchDoneSnackBar(MainActivityBinding mBinding) {
        this.mBinding = mBinding;
    }

    public void show() {
        Snackbar snackbar = Snackbar.make(mBinding.mainLayout, "검색이 완료되었습니다.", Snackbar.LENGTH_LONG);
        snackbar.setAction("확인", v -> {
            snackbar.dismiss();
        });
        snackbar.show();
    }
}

File: app/src/main/java/pe/andy/bookholic/ui/SearchDoneSnackBar.kt
package pe.andy.bookholic.ui

import com.google.android.material.snackbar.Snackbar
import pe.andy.bookholic.databinding.FragmentSearchBinding

class SearchDoneSnackBar(private val mBinding: FragmentSearchBinding) {

    fun show() {
        lateinit var snackbar: Snackbar
        snackbar = Snackbar
                .make(mBinding.layout, "검색이 완료되었습니다.", Snackbar.LENGTH_LONG)
                .setAction("확인") { snackbar.dismiss() }
        snackbar.show()
    }
}
